builder/anka/communicator.go

```diff
@@ -44,51 +46,43 @@ func (c *Communicator) Start(ctx context.Context, remote *packer.RemoteCmd) erro
 		if err != nil {
 			log.Printf("Runner exited with error: %v", err)
 		}
+
 		remote.SetExited(exitCode)
 	}()
 
 	return nil
-
-}
-
-func (c *Communicator) findFUSE() error {
-	notFound, _ := c.Client.Run(client.RunParams{
-		VMName:  c.VMName,
-		Command: []string{"kextstat | grep \"com.veertu.filesystems.vtufs\" &>/dev/null"},
-	})
-	return notFound
 }
 
+// Upload uploads the source file to the destination
 func (c *Communicator) Upload(dst string, src io.Reader, fi *os.FileInfo) error {
 	log.Printf("Uploading file to VM: %s", dst)
 
-	if c.Config.UseAnkaCP == false {
+	if !c.Config.UseAnkaCP {
```

Comment:
We repeat this logic block in a number of places. Can we extract it to a smaller helper?